Remotion/Web/Core/Utilities/HtmlUtility.cs

```diff
@@ -29,25 +28,28 @@ public static string Format (string htmlFormatString, params object[] nonHtmlPar
     {
       string[] htmlParameters = new string[nonHtmlParameters.Length];
       for (int i = 0; i < nonHtmlParameters.Length; ++i)
-        htmlParameters[i] = HtmlEncode (nonHtmlParameters[i].ToString());
+        htmlParameters[i] = WebString.CreateFromText (nonHtmlParameters[i].ToString() ?? string.Empty).ToString (WebStringEncoding.HtmlWithTransformedLineBreaks);
```

Comment:
Why, just why may ToString() return null. This is just...seriously?

Anyhow, since you requested to allow null-input, this solved itself.